### PR TITLE
initial commit of ansible_tower role

### PR DESCRIPTION
**Pull request by robled**

### CODE REVIEW

**Comment by greg-hellings** [comment]
This build failure - is it indicating you might need the EPEL package in CentOS?

File: roles/ansible_tower/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+ansible\_tower
+===========
+
+Deploys a single Ansible Tower instance with a co-located Postgres database
+server
```
Comment (greg-hellings) [inline roles/ansible_tower/README.md:5]: Do we want to be more indicative of how it does this? E.g. by downloading some other automation and running that?

**Comment by robled** [comment]
> This build failure - is it indicating you might need the EPEL package in CentOS?

The builds are passing now without making any changes for CentOS, so ¯\\_(ツ)_/¯ ?

File: roles/ansible_tower/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+    - name: Install Ansible Tower and provide output
+      block:
+        - name: Install Ansible Tower
+          command: >
+            ansible-playbook
+            -i {{ __ansible_tower_installer_root }}/inventory
+            {{ __ansible_tower_installer_root }}/install.yml
+            {% if ansible_tower_extra_vars is defined %}
+            -e {% raw %}"{% endraw %}{% for key, value in ansible_tower_extra_vars.items() %}
```
Comment (greg-hellings) [inline roles/ansible_tower/tasks/main.yml:59]: Why are these quotation marks guarded by {% raw %} statements?

**Review by greg-hellings — APPROVED** [review]

File: roles/ansible_tower/meta/main.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  author: David Roble
+  description: |-
+    Deploy Ansible Tower
+  company: Red Hat, Inc.
+  license: GPLv3
+  min_ansible_version: 2.8
+  platforms:
+    - name: EL
+      versions:
+        - 8
```
[the remark this answers is not in the document]
Comment (robled) [reply roles/ansible_tower/meta/main.yml:11]: Right now I am unable to subscribe my RHEL 7 images. We currently don't have a requirement for RHEL 7 support.  I can dig into the subscription issues if someone has a use case for RHEL 7 specifically.

File: roles/ansible_tower/README.md [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+        job_id: "{{ _ansible_tower_job.id }}"
+        timeout: "{{ ansible_tower_job_timeout }}"
+        <<: *client_config
+      register: _ansible_tower_job_status
+      delegate_to: localhost
+
+    - name: Display job status
+      debug:
+        msg: "{{ _ansible_tower_job_status }}"
+      delegate_to: localhost
```
[the remark this answers is not in the document]
Comment (robled) [reply roles/ansible_tower/README.md:179]: A _job_status_ role perhaps?

File: roles/ansible_tower/tasks/main.yml (hunk 2 above)
Comment (robled) [reply roles/ansible_tower/tasks/main.yml]: Those were left over from a different data structure I was using.  They are no longer necessary, so I've removed them.

**Pull request merged**